### PR TITLE
fix dockerimage workflow

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?

Fix dockerimage workflow. Remove caching.

### What issues does this PR fix or reference?

We have this kind of error in the master branch:
```
...
failed to solve: rpc error: code = Unknown desc = failed to load cache: failed to copy: httpReaderSeeker: failed open: server message: invalid_token: authorization failed
##[error]Process completed with exit code 1.
```

### CODE REVIEW

**Review by sleshchenko — COMMENTED** [review]
Is it a fix of workaround? Maybe we have expired credentials? Did it work?

**Comment by olexii4** [comment]
1. yes, it is a workaround. But I think caching is not sow necessary there.
2. I do not know how to test it manually.

**Comment by akurinnoy** [comment]
@olexii4 
to test the workflow in this PR temporary replace `push:` with `pull_request:` on this line https://github.com/che-incubator/che-dashboard-next/blob/2fbb815db497ab1f62268f75a06d4af2c33427ea/.github/workflows/dockerimage.yml#L12

**Comment by olexii4** [comment]
@akurinnoy Many thanks

**Review by akurinnoy — APPROVED** [review]

**Comment by akurinnoy** [comment]
@olexii4 remember to revert the branch name before merge.

**Review by sleshchenko — APPROVED** [review]

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ name: Next Dockerimage
 
 on:
   push:
-    branches: [ master ]
+    branches: [ dockerimage_workflow ]
```
Comment (sleshchenko) [inline .github/workflows/dockerimage.yml]:
```suggestion
    branches: [ master ]
```

**Pull request merged**